server/utils/helpers/activityTimeSlotMappers.ts

```diff
@@ -344,6 +344,89 @@ export function calculateUniqueSlots(slotsA: Slot[], slotsB: Slot[]): Slot[] {
     .filter(s => s.daysOfWeek.length > 0)
 }
 
+export function calculateExclusionSlots(exclusions: Slot[], updatedExclusions: Slot[]): Slot[] {
+  const addedSlots: Slot[] = []
+
+  exclusions.forEach(exclusion => {
+    const filteredExclusions = updatedExclusions.filter(
+      e => e.timeSlot === exclusion.timeSlot && e.weekNumber === exclusion.weekNumber,
+    )
+    const days: ('MONDAY' | 'TUESDAY' | 'WEDNESDAY' | 'THURSDAY' | 'FRIDAY' | 'SATURDAY' | 'SUNDAY')[] =
```

Comment:
Could we have ('MONDAY' | 'TUESDAY' | 'WEDNESDAY' | 'THURSDAY' | 'FRIDAY' | 'SATURDAY' | 'SUNDAY')[] as a type somewhere so it can be reused in other places?

Reply:
will do

Reply:
Or failing that `const days: Slot['daysOfWeek'] =` ?

Reply:
I've added a type, sp  const days: DaysOfWeek[]